server/command_hooks.go

```diff
@@ -80,12 +79,12 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 
 		return &model.CommandResponse{
 			ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL,
-			Text:         "Disabled demo plugin hooks.",
+			Text:         p.Helpers.TContext(c, "command.disabled"),
 		}, nil
 	}
 
 	return &model.CommandResponse{
 		ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL,
-		Text:         fmt.Sprintf("Unknown command action: " + args.Command),
+		Text:         p.Helpers.TContext(c, "command.unknown_command", map[string]interface{}{"Command": args.Command}),
```

Comment:
I guess there's no easier way to send the args?

Reply:
It's how it's done on the server. Keeping the same API to be consistent. 
It has been proposed alternatively to forget what the server does and upgrade to v2 of the i18n library we use: https://github.com/nicksnyder/go-i18n
Thoughts?

Reply:
FIY: I commented about this topic in the server PR https://github.com/mattermost/mattermost-server/pull/10949#issuecomment-497027177